client/src/screens/AdminPage.js

```diff
@@ -39,10 +39,10 @@ export const AdminPage = () => {
 		setPetState(petState.filter((pet) => pet.id !== id));
```

Comment:
We should probably move this inside the axios promise so that we only delete when we get a OK response from the DB.